DigitalLearningSolutions.Data/Models/CourseDelegates/CourseDelegateForExport.cs

```diff
@@ -1,35 +1,35 @@
-﻿namespace DigitalLearningSolutions.Data.Models.CourseDelegates
-{
-    public class CourseDelegateForExport : CourseDelegate
-    {
-        public string? Answer1 { get; set; }
-
-        public string? Answer2 { get; set; }
-
-        public string? Answer3 { get; set; }
-
-        public string? Answer4 { get; set; }
-
-        public string? Answer5 { get; set; }
-
-        public string? Answer6 { get; set; }
-
-        public string? CourseAnswer1 { get; set; }
-
-        public string? CourseAnswer2 { get; set; }
-
-        public string? CourseAnswer3 { get; set; }
-
-        public int LoginCount { get; set; }
-
-        public int Duration { get; set; }
-
-        public int? DiagnosticScore { get; set; }
-
-        public string?[] CustomRegistrationPromptAnswers =>
-            new[] { Answer1, Answer2, Answer3, Answer4, Answer5, Answer6 };
-
-        public string?[] CustomAdminFieldAnswers =>
-            new[] { CourseAnswer1, CourseAnswer2, CourseAnswer3 };
-    }
-}
+﻿namespace DigitalLearningSolutions.Data.Models.CourseDelegates
+{
+    public class CourseDelegateForExport : CourseDelegate
+    {
+        public string? Answer1 { get; set; }
+
+        public string? Answer2 { get; set; }
+
+        public string? Answer3 { get; set; }
+
+        public string? Answer4 { get; set; }
+
+        public string? Answer5 { get; set; }
+
+        public string? Answer6 { get; set; }
+
+        public string? CourseAnswer1 { get; set; }
+
+        public string? CourseAnswer2 { get; set; }
+
+        public string? CourseAnswer3 { get; set; }
+
+        public int LoginCount { get; set; }
+
+        public int Duration { get; set; }
+
+        public int? DiagnosticScore { get; set; }
+
+        public string?[] CentreRegistrationPromptAnswers =>
+            new[] { Answer1, Answer2, Answer3, Answer4, Answer5, Answer6 };
+
+        public string?[] CourseAdminFieldAnswers =>

```

Comment:
changed :)

Reply:
I don't think the new names are confusing, so don't feel you've got to make more changes, but I think having `Answers` there was useful - I didn't mean that they should be called `DelegateRegistrationPrompts` etc, just just they were to do with the delegate rather than the centre/course. I'll try and be consistent with my comments and use `this font` when suggesting a specific name.